testsupport/tiers/checks.go

```diff
@@ -271,6 +275,57 @@ func (a *testTierChecks) GetClusterObjectChecks() []clusterObjectsCheck {
 	return []clusterObjectsCheck{}
 }
 
+type appstudioTierChecks struct {
+	baseTierChecks
+}
+
+func (a *appstudioTierChecks) GetTierObjectChecks() []tierObjectCheck {
+	return []tierObjectCheck{nsTemplateTier(a.tierName, 30)}
+}
+
+func (a *appstudioTierChecks) GetNamespaceObjectChecks(nsType string) []namespaceObjectsCheck {
+	checks := append(commonChecks,
+		limitRange(defaultCPULimit, "750Mi", "10m", "64Mi"),
+		userViewRoleBinding(),
+		crtadminPodsRoleBinding(),
+		crtadminViewRoleBinding(),
+		appstudioEditRoleBinding(),
+		userSaReadRoleBinding(),
+		execPodsRole(),
+		toolchainSaReadRole(),
+		appstudioServiceAccount(),
+		numberOfToolchainRoles(2),
+		numberOfToolchainRoleBindings(5),
+		numberOfToolchainServiceAccounts(1),
+	)
+
+	checks = append(checks, commonNetworkPolicyChecks()...)
```

Comment:
`commonNetworkPolicyChecks` doesn't contain the networkPolicy for CRW. I should also check the number of present networkPolicies
```suggestion
	checks = append(checks, append(commonNetworkPolicyChecks(), networkPolicyAllowFromCRW(), numberOfNetworkPolicies(6))...)
```